frontend_lib/src/component/Button/IconButton.jsx

```diff
@@ -23,6 +23,7 @@ const IconButton = props => {
       style={style}
       onClick={props.onClick}
       disabled={props.disabled}
+      title={props.title}
```

Comment:
```suggestion
      title={props.title || props.text}
```
Like this we always have a tooltip.